bin/varnishtest/tests/s00002.vtc

```diff
@@ -43,7 +43,7 @@ varnish v1 -vcl {
 		.port = "${s1_port}";
 		.probe = {
 			.url = "/";
-			.timeout = 30ms;
+			.timeout = 1s;
```

Comment:
why this?

Reply:
This change can be dropped.

We saw the probe time out because the probe worker would refresh the VCL and sometimes see `vcl_active` as NULL. That would put the worker to sleep for 100ms, and as the `VCL_refresh()` happened within the timed codepath, time out the probe in this test case. As the `VCL_refresh()` added in this changeset happens outside of the timed codepath (and under debug flag), this is not be an issue.